### DISCUSSION TITLE
トラックボールの感度変更について

### DISCUSSION THREAD

**Discussion by Y-KBYM** [Yowkees/keyball#352] [Q&A]
トラックボールの感度変更は、例えばCPIを1000増加させるには、0x7e02にアクセスするよう、キーコード一覧に書いてあります。

https://github.com/Yowkees/keyball/blob/main/qmk_firmware/keyboards/keyball/lib/keyball/keycodes.md#japanese　より
例：CPI_I100 | Kb 2 | 0x7e02 | CPIを100増加させます(最大:12000)

しかしながら、デフォルトFWのkeyball39_viaでは、レイヤ3に割り当てられている0x5DA8がそれと同じ機能を持っています。

0x7e02にアクセスしても何も反応がないのですが、これはどちらが正しいのでしょうか。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
Keyball用の特殊キーコードはビルドするQMKファームウェアのバージョンによってズレることがあります。
現在、最新0.22のQMKに対応したKeyball用ファームウェアを作成しておりますので、最新バージョンリリース後に見直したいと思いますので、お願いいたします。

**Reply by Y-KBYM** [reply]
Yowkees様
承知しました。ありがとうございました。
